src/ceph-volume/plugin/zfs/ceph_volume_zfs/main.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+import argparse
+import os
+import pkg_resources
+import sys
+import logging
+
+import ceph_volume_zfs
+import main
+from ceph_volume import log, conf, configuration
+from ceph_volume import exceptions
+from ceph_volume import terminal
+from ceph_volume_zfs import devices
+
+if __name__ == '__main__':
+    main.ZFSVOL()
+  
+
+class ZFSVOL(object):
+
+    help_menu = 'Deploy OSDs with ZFS'
+    _help = """
+Use ZFS as the underlying technology for OSDs
+
+--verbose   Increase the verbosity level
+    """
+    name = 'zfs'
+
+    def __init__(self, argv=None, parse=True):
+        self.mapper = {
+            'zfs': devices.zfs.ZFS,
+        }
+	if argv is None:
+            self.argv = sys.argv
+        else:
+            self.argv = argv
+        if parse:
+            self.main(self.argv)
+	print(self.argv)
+
+    def help(self, warning=False):
+	warning = 'See "ceph-volume-zfs --help" for full list of options.' if warning else ''
```

Comment:
is `ceph-volume-zfs` supposed to be launched by `ceph-volume`?

Reply:
@tchaikov 
I also have bin/ceph-volume-zfs, which suggests that it can be called standalone.
But still you need to call it like `ceph-volume-zfs zfs ...` which is sort redundant.
It was conveiniant during testing.
Could be fixed, but perhaps there should not even be a ceph-volume-zfs?